core/src/main/resources/lib/audio.fjs

```diff
@@ -95,6 +88,11 @@ export default (componentType: ComponentType) => ({ ophan, dom, viewport }: Serv
     });
   };
 
+  const onDurationChange = () => {
```

Comment:
Can we tie the scrubber seek calculation to this value too? It currently reads a .getAttribute('data-duration') which can be incorrect due to ads. It'd be nice to kill that issue now if we can.

Reply:
Oooooh yeah you're right. will do 👍

Reply:
Done. I will test as soon as the previewer has been approved.